### PR TITLE
Fix source stability

### PR DESCRIPTION
**Pull request by kfurmann**
stabilized blueprints spec

### CODE REVIEW

File: src/components/ui/common/Source.js [hunk 1]
```diff
@@ -43,7 +43,20 @@ exports.getLoadingMessage = function() {
     //logger.info('getting loading message');
     var locator = by.css('.noPreview p');
     browser.driver.wait(function () {
+        console.log('waiting...');
```
Comment (guy-mograbi-at-gigaspaces) [inline src/components/ui/common/Source.js]: pls remove this..

File: src/components/ui/common/Source.js [hunk 2]
```diff
@@ -43,7 +43,20 @@ exports.getLoadingMessage = function() {
...
         return browser.driver.isElementPresent(locator);
-    }, 10000);
-    return browser.driver.findElement(locator).getText(); // use selenium API directly since we don't want angular synced here..
+
+    }, 40000);
+
+    var result = '';
+    browser.driver.wait(function(){
+        return browser.driver.findElement(locator).getText().then(function(text){
+            console.log('text is [' + text + ']');
```
Comment (guy-mograbi-at-gigaspaces) [inline src/components/ui/common/Source.js]: pls remove this

File: spec/sanity/blueprints.js [hunk 3]
```diff
@@ -40,8 +42,22 @@ describe('blueprints page', function(){
...
+            browser.get('/#/blueprint/nodecellar1/source');
+            expect(BlueprintPage.Source.getLoadingMessage()).toBe('Generating Blueprint Source View...');
+
+            browser.sleep(0).then(function(){
+                browser.ignoreSynchronization = false;
+                done();
+            });
+        });
+
+        it('should not ignore sync', function(){
```
Comment (guy-mograbi-at-gigaspaces) [inline spec/sanity/blueprints.js]: we don't need this it anymore..

File: spec/sanity/blueprints.js [hunk 4]
```diff
@@ -13,23 +14,24 @@ describe('blueprints page', function(){
...
+    });
 
-        it('should have working source section', function(done) {
+    describe('source view', function(){
 
-            logger.trace('start blueprint source section test');
+        it('should have working source section', function(done) {
+            components.ui.blueprints.IndexPage.goToBlueprint({'name': testConf.blueprints.blueprintToRead});
             BlueprintPage.goToSource();
-            expect(BlueprintPage.Source.getLoadingMessage()).toBe('Generating Blueprint Source View...');
```
Comment (edenpessach) [inline spec/sanity/blueprints.js]: By removing the expect this test, does not check anything...

Comment (guy-mograbi-at-gigaspaces) [inline spec/sanity/blueprints.js]: There is an implicit expect under 'getFileTitle'.. it should externalized to the test for readability. we sometimes assert stuff during internal functions like 'goToBlueprint' we expect to have such a blueprint, to make debug easier when test fails. but in this scenario it feels forced.

**Pull request merged**